### PR TITLE
Center FAQ and Getting started

### LINKED ISSUE #44: Center FAQs
The FAQ section is left aligned. It was discussed that we need to update it so the text is aligned to the center

### LINKED ISSUE #81: Center getting started section
As per Zeplin here https://app.zeplin.io/project/5d5e913bc8e2bb9b4eff9321/screen/5e9016025f5a51babb2fbe65/

### PR DESCRIPTION
**Pull request by sradevski**
Fixes #81
Fixes #44

Change-type: patch
Signed-off-by: Stevche Radevski <stevche@balena.io>

### CODE REVIEW

File: src/components/GetStarted.tsx [hunk 1]
```diff
@@ -197,7 +205,7 @@ const GetStarted = ({
 						>
 							{deviceTypeSelector}
 							{intro}
-							<Heading.h3 bold my={3}>
+							<Heading.h3 bold mb={3} mt={4}>
```
Comment (iamsolankiamit) [inline src/components/GetStarted.tsx:220]: need to update this in `laptop and computer` tab as well.

Comment (sradevski) [reply src/components/GetStarted.tsx:220]: nice catch 👍

File: src/components/Guides.tsx [hunk 2]
```diff
@@ -201,8 +201,8 @@ const genericGuide: any = {
 	) => ({
 		intro: renderGenericInfo(deviceType?.name),
 		steps: [
-			renderDownloadOS(sdk, app, deviceType),
 			renderDownloadEtcher(),
+			renderDownloadOS(sdk, app, deviceType),
```
Comment (iamsolankiamit) [inline src/components/Guides.tsx]: download os should be the first step, @chrisys updated this earlier today.

Comment (sradevski) [reply src/components/Guides.tsx]: I saw in the designs it's etcher :/ cool thanks

**Comment by sradevski** [comment]
@balena-ci rebase

File: src/components/GetStarted.tsx [hunk 3]
```diff
@@ -226,7 +234,7 @@ const GetStarted = ({
 						>
 							{deviceTypeSelector}
 							{intro}
-							<Heading.h3 bold my={3}>
+							<Heading.h3 bold my={3} mt={4}>
```
Comment (iamsolankiamit) [inline src/components/GetStarted.tsx]: mb={3} here?

**Review by iamsolankiamit — APPROVED** [review]

**Pull request merged**